google/cloud/bigtable/examples/bigtable_samples.cc

```diff
@@ -302,6 +302,22 @@ void WaitForConsistencyCheck(google::cloud::bigtable::TableAdmin admin,
 }
 //! [wait for consistency check]
 
+//! [check consistency]
+void CheckConsistency(google::cloud::bigtable::TableAdmin admin,
+                      std::string const& table_id_param) {
+  google::cloud::bigtable::TableId table_id(table_id_param);
+  google::cloud::bigtable::ConsistencyToken consistency_token(
+      admin.GenerateConsistencyToken(table_id.get()));
+  auto result = admin.CheckConsistency(table_id, consistency_token);
+  if (result) {
+    std::cout << "Table is consistent" << std::endl;
+  } else {
+    std::cout << "Table is not consistent" << std::endl;
```

Comment:
Please change this example to receive the consistency token as a parameter (it is basically a string).  And change the output here to "Table is not yet consistent, try again later with the same token."